mocha-client-tests/README.md

```diff
@@ -0,0 +1,13 @@
+#Mocha client tests for WebExtensions
+##Install dependency
+`npm install` and `bower install`  
+
+And inside of addon please run `npm i`
```

Comment:
I think it would be more helpful to spell out `npm install` in all cases since not all readers will know about the `i` alias.